data/src/isupport.rs

```diff
@@ -0,0 +1,614 @@
+// ISUPPORT Parameter References
+// - https://defs.ircdocs.horse/defs/isupport.html
+// - https://modern.ircdocs.horse/#rplisupport-005
+// - https://ircv3.net/specs/extensions/chathistory
+// - https://ircv3.net/specs/extensions/monitor
+// - https://ircv3.net/specs/extensions/utf8-only
+// - https://ircv3.net/specs/extensions/whox
+// - https://github.com/ircv3/ircv3-specifications/pull/464/files
+#[allow(non_camel_case_types)]
+#[derive(Debug)]
+pub enum Parameter {
+    ACCEPT(u16),
+    ACCOUNTEXTBAN(Vec<String>),
+    AWAYLEN(Option<u16>),
+    BOT(char),
+    CALLERID(char),
+    CASEMAPPING(CaseMap),
+    CHANLIMIT(Vec<ChannelLimit>),
+    CHANMODES(Vec<ChannelMode>),
+    CHANNELLEN(u16),
+    CHANTYPES(Option<String>),
+    CHATHISTORY(u16),
+    CLIENTTAGDENY(Vec<ClientOnlyTags>),
+    CLIENTVER(u16, u16),
+    CNOTICE,
+    CPRIVMSG,
+    DEAF(char),
+    ELIST(String),
+    ESILENCE(Option<String>),
+    ETRACE,
+    EXCEPTS(char),
+    EXTBAN(Option<char>, String),
+    FNC,
+    HOSTLEN(u16),
+    INVEX(char),
+    KEYLEN(u16),
+    KICKLEN(u16),
+    KNOCK,
+    LINELEN(u16),
+    MAP,
+    MAXBANS(u16),
+    MAXCHANNELS(u16),
+    MAXLIST(Vec<ModesLimit>),
+    MAXPARA(u16),
+    MAXTARGETS(Option<u16>),
+    METADATA(Option<u16>),
+    MODES(Option<u16>),
+    MONITOR(Option<u16>),
+    MSGREFTYPES(Vec<MessageReferenceType>),
+    NAMESX,
+    NETWORK(String),
+    NICKLEN(u16),
+    OVERRIDE,
+    PREFIX(Vec<PrefixMap>),
+    SAFELIST,
+    SECURELIST,
+    SILENCE(Option<u16>),
+    STATUSMSG(String),
+    TARGMAX(Vec<CommandTargetLimit>),
+    TOPICLEN(u16),
+    UHNAMES,
+    USERIP,
+    USERLEN(u16),
+    UTF8ONLY,
+    VLIST(String),
+    WATCH(u16),
+    WHOX,
+    Negation(String),
```

Comment:
`Negation` isn't a parameter, it's an operation. Let's have `isupport::Operation::Add(Parameter) / Remove(String)`.

Reply:
Great suggestion, thank you!  Implemented.